### PR TITLE
Fireauth vs fusionauth blog post

### PR DESCRIPTION
**Pull request by mooreds**
Guest blog post from Hunter

### CODE REVIEW

File: site/_posts/2020-05-23-goodbye-google-firebase-auth.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+broader appeal to those who want to get started with authentication, FusionAuth
+works on finishing what Google Firebase Authentication began.
+
+I enjoy the sleek interface of FusionAuth and the ability to easily see the most
+recent logins on the main page; including a graph representing the logins by hour.
+The [reporting and metrics](/docs/v1/tech/apis/reports) of FusionAuth make analyzing your authentication
+platform a breeze. 
+
+{% include _image.liquid src="/assets/img/blogs/fireauth-vs-fusionauth/reporting.png" alt="The reporting dashboard." class="img-fluid" figure=false %}
+
```
Comment (dpolstra) [inline site/_posts/2020-05-23-goodbye-google-firebase-auth.md:91]: There should be an H2 here. Not sure what it is

File: site/_posts/2020-05-23-goodbye-google-firebase-auth.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+authentication of the web hooks, and setup user actions based on web hook
+execution states.
+
+{% include _image.liquid src="/assets/img/blogs/fireauth-vs-fusionauth/webhook.png" alt="Webhook configuration lets you integrate easily with other applications." class="img-fluid" figure=false %}
+
+## Enhanced Security
+
+Identity providers are important, but vary depending on your specific needs. Being
+able to have third party logins, through Google Sign In, FaceBook, Twitter.
+can enhance the ease of signing up users to your application. Both Google Fire
```
Comment (dpolstra) [inline site/_posts/2020-05-23-goodbye-google-firebase-auth.md:114]: Both Google Fire Authentication and FusionAuth Support

File: site/_posts/2020-05-23-goodbye-google-firebase-auth.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+Authentication support a variety of [identity providers](/docs/v1/tech/identity-providers/); what sets FusionAuth apart is
+their integration with [HYPR](/docs/v1/tech/identity-providers/hypr).
+
+HYPR is a passwordless Multi-Factored-Authentication (MFA) that greatly speeds
+up passwordless authentication. It runs silently in the background and makes it
+easy for your users to enroll multiple mobile devices with their desktops and
+laptops.
+
+## Reactor
```
Comment (dpolstra) [inline site/_posts/2020-05-23-goodbye-google-firebase-auth.md:123]: FusionAuth Reactor

File: site/_posts/2020-05-23-goodbye-google-firebase-auth.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
+---
+layout: blog-post
+title: Goodbye Google Firebase Authentication
```
[the remark this answers is not in the document]
Comment (dpolstra) [reply site/_posts/2020-05-23-goodbye-google-firebase-auth.md:3]: Or FusionAuth vs Google Firebase Authentication. Why I said goodbye to Firebase.

**Comment by mooreds** [comment]
Closing as we're not going to use this post.

**Pull request closed without merging**